vpr/src/device/rr_graph_builder_view.h

```diff
@@ -0,0 +1,53 @@
+#ifndef _RR_GRAPH_BUILDER_VIEW_
+#define _RR_GRAPH_BUILDER_VIEW_
+
+#include <exception>
+
+#include "rr_graph_fwd.h"
+#include "rr_node_fwd.h"
+#include "rr_graph2.h"
+#include "vtr_log.h"
+#include "vtr_memory.h"
+#include "vpr_utils.h"
+#include "vtr_strong_id_range.h"
+#include "vtr_array_view.h"
+#include "rr_graph_storage.h"
+#include "rr_graph_view.h"
```

Comment:
Below, the types `rr_graph_storage`, `rr_node_indices`, `RRNodeId`, `t_rr_type` and `e_side` are used. Do we have to include all the above to get access to these ?

IWYU, but not more.

Reply:
Agree. Removed all the unused headers.